README.md

```diff
@@ -170,6 +170,24 @@ The class attribute of the 'small' element will mirror the class attribute of th
 
 If the `html_safe_errors: true` option is specified on a field, then any HTML you may have embedded in a custom error string will be displayed with the html_safe option.
 
+### Prefix and Postfix
+Simple prefix and postfix span elements can be added beside inputs.
+```ruby
+f.text_field :name, prefix { value: 'foo' small: 2, large: 3 }
```

Comment:
I've added it in this PR https://github.com/sgruhier/foundation_rails_helper/pull/109